docs/Cluster-VictoriaMetrics.md

```diff
@@ -712,7 +712,9 @@ Some workloads may need fine-grained resource usage limits. In these cases the f
   may exceed the per-query memory limit by a small percent. The total memory limit for concurrently executed queries can be estimated
   as `-search.maxMemoryPerQuery` multiplied by `-search.maxConcurrentRequests`.
 - `-search.maxUniqueTimeseries` at `vmselect` component limits the number of unique time series a single query can find and process.
-  `vmselect` passes the limit to `vmstorage` component, which keeps in memory some metainformation about the time series located
```

Comment:
The same changes needs to be mentioned at https://docs.victoriametrics.com/#resource-usage-limits